tests/stdlib/tmath.nim

```diff
@@ -350,5 +348,13 @@ template main =
     doAssert copySign(-NaN, 0.0).isNaN
     doAssert copySign(-NaN, -0.0).isNaN
 
+    when nimvm:
+      discard
+    else:
+      when not defined(js):
+        doAssert copySign(-1.0, -NaN) == 1.0
```

Comment:
until  https://github.com/timotheecour/Nim/issues/499 is fixed, this test is meaningless:
```nim
        doAssert copySign(-1.0, -NaN) == 1.0
        doAssert copySign(-1.0, NaN) == 1.0
```
both pass currently, which is not correct behavior IMO

IMO `doAssert copySign(-1.0, NaN) == 1.0` is probably correct (we can check with signbit once we have it), but `doAssert copySign(-1.0, -NaN) == 1.0` is probably not correct, and should be:
```nim
when false: # xxx bug
  doAssert copySign(-1.0, -NaN) == -1.0
```

likewise with `doAssert copySign(10.0, -NaN) == 10.0`